spryx_http/base.py

```
@@ -78,7 +78,12 @@ def _extract_data_from_response(self, response_json: ResponseJson | None) -> Any
         Returns:
             Any: The response data.
         """
-        return response_json
+        if response_json:
+            data = response_json.get("data", response_json)
+            return data
+            
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)